src/__tests__/__snapshots__/to-have-style.tsx.snap

```diff
@@ -6,15 +6,26 @@ exports[`.toHaveStyle handles negative test cases 1`] = `
 - Expected
 
   backgroundColor: blue;
-- transform: [{"scale": 1}];
-+ transform: [{"scale": 2}];"
+  transform: [
```

Comment:
i had the doubt when doing the change, do we want to output everything or just the result which is different than the expectation?

Reply:
The idea behind this is that particular style attributes are not divisible, they need to match as units.

Normally this is not an issue as most style attributes are `string`, `number`, etc. The only two `Array` attributes are `transform` and `fontVariant` and we want to output the whole array in this case.